src/sentry/static/sentry/app/views/performance/transactionDetails/eventMetas.tsx

```diff
@@ -26,9 +27,8 @@ type Props = {
 };
 
 function EventMetas({event, organization, projectId, location, quickTrace}: Props) {
-  if (!isTransaction(event)) {
-    return null;
-  }
+  const type = isTransaction(event) ? 'transaction' : 'event';
+  const transaction = event as EventTransaction;
```

Comment:
Is this explicit casting necessary? If necessary, you might be able to change `type === transaction` below to `isTransaction(event)` and Typescript might be able to pick up its an `EventTransaction`.

Reply:
I had that originally but it kept beeping about the timestamps not existing, is there any other way to do `if event is EventTransaction`?